ci/properties/grunt.properties.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Grunt",
+    "description": "Build a Grunt project with npm.",
```

Comment:
`Build a NodeJS project with npm and grunt`

Reply:
updated